lib/web_ui/lib/src/engine/dom_renderer.dart

```diff
@@ -237,8 +215,8 @@ class DomRenderer {
   void reset() {
     _styleElement?.remove();
     _styleElement = html.StyleElement();
-    html.document.head.append(_styleElement);
-    final html.CssStyleSheet sheet = _styleElement.sheet;
+    html.document.head!.append(_styleElement!);
```

Comment:
You are construction _styleElement a line above why do you need _styleElement! here?

Reply:
It's because it's a field. This method is called once during start-up, so the extra null check doesn't hurt performance. We can clean up later (just so we have fewer potential NPEs in the code).